### PR TITLE
README update : dependencies

### LINKED ISSUE #296: Add graphviz as a requirement in the README, as we need it to run 'dot'
`dot` is needed to display the dot graphs and visuals in the dataflow preview and steps pages.

https://graphviz.org/doc/info/lang.html

Installation instructions here: https://graphviz.org/download/ (linux, windows, etc)

### PR DESCRIPTION
**Pull request by marcghaly**
Just a small readme update

This resolves #296

### CODE REVIEW

**Comment by keevan** [comment]
Thanks @marcghaly, can you link the relevant issue as well please? I think it's a good idea to include a link to the official docs as well, for people who don't use `apt`

**Comment by marcghaly** [comment]
Hi @keevan, I have added a link to documentation, the issue is similar to this one https://serverfault.com/questions/575239/error-unable-to-find-dot-command-of-the-graphviz-package

Screenshot from 2022-07-15 20-47-29

Regards,

Marc

File: README.md [hunk 1]
```diff
@@ -48,3 +48,12 @@ From Moodle siteroot:
 git clone git@github.com:catalyst/moodle-tool_dataflows.git admin/tool/dataflows
 ```
 
+## Dependencies
+
+Make sure graphviz PHP module is installed
```
Comment (keevan) [inline README.md:53]: I don't think graphviz is a PHP module, as it's a standalone thing. Perhaps you can remove this.

```suggestion
Make sure graphviz is installed
```

**Review by keevan — COMMENTED** [review]
> Hi @keevan, I have added a link to documentation, the issue is similar to this one https://serverfault.com/questions/575239/error-unable-to-find-dot-command-of-the-graphviz-package
> 
> Screenshot from 2022-07-15 20-47-29
> 
> Regards,
> 
> Marc

Perhaps you can also add a check for this, so it displays - in addition - a link in the UI to the dependencies section of the dataflows plugin so the user can view and install what's needed for `dot`. That might be a more friendly UX than simply showing the error message.

The rest of the changes to the README look good. 

Thanks @marcghaly :+1:

**Comment by keevan** [comment]
Split remaining feedback item into a new issue:

https://github.com/catalyst/moodle-tool_dataflows/issues/315

**Pull request merged**